packages/generators/package.json

```diff
@@ -25,21 +25,23 @@
     "plugin-template"
   ],
   "dependencies": {
+    "@types/inquirer": "^9.0.7",
+    "@yeoman/types": "^1.2.0",
+    "mem-fs": "^4.1.0",
+    "webpack": "^5.91.0",
     "webpack-cli": "^5.1.4",
-    "yeoman-environment": "^3.9.1",
-    "yeoman-generator": "^5.7.0"
+    "yeoman-environment": "^4.4.0",
+    "yeoman-generator": "^7.2.0"
```

Comment:
There are no breaking changes?

Reply:
I'd read the changelogs from yeoman and maybe even the source. From experience, their version updates have a lot of changes that might break our implementation